source/bson-binary-vector/tests/packed_bit.json

```diff
@@ -84,15 +78,6 @@
       "dtype_hex": "0x10",
       "dtype_alias": "PACKED_BIT",
       "padding": -1
-    },
-    {
-      "description": "Vector with float values PACKED_BIT",
-      "valid": false,
-      "vector": [127.5],
-      "dtype_hex": "0x10",
-      "dtype_alias": "PACKED_BIT",
-      "padding": 0
     }
   ]
 }
-
```

Comment:
And this one? Passing a float when packed bit requires an int (or unit) in [0,127]?

Reply:
Another duplicate of L57-L63